src/core/CircularProgress/index.js

```diff
@@ -2,17 +2,14 @@
 
 import React from 'react'
 import PropTypes from 'prop-types'
-import { bemClassNames } from '../../utils'
-import './styles.css'
-
-const bem = bemClassNames('circular-progress')
+import s from './styles'
 
 function CircularProgress({ size, center, dashed }) {
     const progressBar = (
-        <div role="progressbar" className={bem.b(size)}>
-            <svg viewBox="22 22 44 44" className={bem.e('viewbox')}>
+        <div role="progressbar" className={s('container', size)}>
+            <svg viewBox="22 22 44 44" className={s('viewbox')}>
                 <circle
-                    className={dashed ? bem.e('dashed') : ''}
+                    className={dashed ? s('dashed') : ''}
```

Comment:
**classnames** should be able to output "dashed" class if you do:

```
className={s({ dashed })}
```

From the docs:

```
classNames('foo', { foo: false, bar: true }); // => 'bar'
```